app/models/barcode_item.rb

```diff
@@ -19,6 +19,19 @@ class BarcodeItem < ApplicationRecord
   validates :quantity, :item, presence: true
   validates :quantity, numericality: { only_integer: true, greater_than_or_equal_to: 0}
 
```

Comment:
Also, I'm also realizing that we have a validation on the value that it has to be present and unique. Do we want to change that and scope the uniqueness to the organization? I wonder if that will work since organization is optional... Time to read.

The thing I'm thinking about is when diaper banks start creating their own barcodes and it doesn't seem unfeasible that two may create the same custom barcode `fortysize3` or something but only the first would be allowed to do it... 

Sorry, I realize this has nothing to do with your pull request it is just me thinking out loud. :)

Reply:
setting the uniqueness to the organization_id seems to work and not break the tests...
ie- ` validates :value, presence: true, uniqueness: { scope: :organization_id }`